### ISSUE TITLE
cant connect framelib objects between embedded bpatchers

### ISSUE THREAD

**Issue by jamesb93** [AlexHarker/FrameLib#70]
I am getting an error about patching contexts when trying to connect FrameLib objects inside an embedded bpatcher with the top level patch.

```
<pre><code>
----------begin_max5_patcher----------
1406.3ocyYsrjhaCEcM7U3xUVxPa4G.N6R1jMIe.o5JEkvV.ZFYIWRx.clp6
u8nG1FYiMCcCyLYCFzUOtmy8kuhuNch+F1Ijv26W8d1axjuNcxDyP5AlT+6I
9EvSYDnvLM+MkPY1dD2elU3AL53Ar.uASvxWzSATKgscq.Is6cv7fY5O79mZ
gDV1WP44b3NQFmQH5oETKKify9hbOmUsau63HJbCAs+xEXEb3RAa1UvxQcFg
wyUJuiZhyM3hs4yeJJpAUrJIAIkuThr5uueqlavOltaMGkUit3EIZ7AreFZ.
apKZoUEXpZCEtphZP6wHbUGTwFTt6.M7s0jTaSl3uEqPLhKvLpyrm3CKKcFd
hyRzFxOyLazpYsCgoLW5PMDGosm10G0NJjqPsTA4JtgS7OsH1+71nnYNsBa1
I6fuNsQkLNOTXARTByrKV6i0H9LOtbwRM0ElXXvEwlGwqbXRsI0553vjJyUI
hhokbj.QkPYeNIGsEVQjq2xnRA9eMp.HbdvPx2VqhCJTCBi9+abLjzhfcbbN
ipUBsvP2gaNtma7NRbAiYFTX44E2pyJ+QEuLhPgBjUhMPtIxif5brRFizUT6
5HnsxZwkXJsGKJYkiKji2s+JqcCSIr3Z6sQhXcE0JcsxoPtV.OzkskPBghNp
Peus+DjhKfRjDaMAgAsBuHu.nqjCCHIW4jmgNhyk1TLtNCpoiKabh7asx43c
Hgr6XRU9qtiHjuXIcmgpZRZtVhJJIJTzcBcRB6Fw5lLty38RJqBsDvcn13Qs
lgNY0UF06W.tRbR3ARcELdRuwR7U6SaeDkZBdCcbvuH2WXWAt4+ZD75zoMeY
1GjNra6XXdo+s.pv.fATAMeNNn.iBp.GIYrhBU1oNVdipQyQmdzLPcHzP9Ca
IyKQ7M5rnuMFEEcmtEfXSBbCCtL3a6V.9g3VbcRYKWYffmdy6IcwLuRHGVHF
if.Oj3ljT6iv+uvPD7AzbCo3bbGf715dCNA25tWBxEl2Fx3JrneXjCkl1MpT
Q9HoJgoMCdeF3RFeVeVejZ8cr5BUN47JBh6odW0wPT+ydbyoKRWdqFSsxTWo
wQcn9iZucSgpKeluFJkb7lJI572DNFaGqs1XRpPrsthck6R8DFc2UM7clsN1
oG15HWrmwk291gTft11pTYf19pdF1wF2YAMNCgmE2FP39UmnjyuGFASGq.rw
bqkOb3ifUwyZ7BaRM3001qdiBIl15L8745P5IdSguuasH5V0hKU2BbdICSk0
DBHLZdhx6N1jSOLt8GemT8zaUyS+dxeouS9aP2qomGw7oUytktr6Tp5bBqRt
xt38mrid+ktu1K6fMLve71TiVXanx1dUpsWqnwZSELXapAONvnp4pZYPUwcH
b.9XchWCwnTaGjgeDHBdnPTYvPbUpq27.AAAdEBumDGwJEWWWyC38j58yCFh
ABuKFHDXeDE8ymB3ncXghEdyKZNHdHr9Qu2kU1bR1WwLc00gZ3OTnpTof.vP
fM99LrVvB.Q+7QqJcDiikXAp4slIriCA4j6KZ11DFH9tgrYccq1aYf90Gp4g
KpKD0ot5H0DzgtM0Dtfqu0SpYSt9I0UcZuayfe.Gc7vGM39N5ja4nAOB98lr
jZ0AbumT7sdR2Ml.2xIEbwIYCK5cyw5Co2MF261hu7lhG+Vh6eCwlaG1dYY8
BEa6Qvl242U8q7o57nM2EaWZ.lko5UKiQrpl9OaHBrbQ7R8+3vhj3zfP82R0
CATly4sfeVM6xUqu+814WnNHbSvjq0nmB92HhJq26VEWEl.LZ3xUfU.yeNRR
ZRbRsBVaBc6h8YSerIoAAqdT.npXCh+GLR96V80jYvBsZmjFBRLHAD.RW7wY
XqW3H+YASec5+AjZMt2O
-----------end_max5_patcher-----------
</code></pre>
```

**Comment by AlexHarker** [comment]
Have you tested this with non-embedded ones? I suspect that the embedding is the issue here - I never use that feature so I'll need to work on detecting it.

**Comment by AlexHarker** [comment]
Fixed on object review in 53afca663491291de4c6e4ee8735f08f8bf0867a
